charts/perses/templates/ingress.yaml

```diff
@@ -23,7 +12,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-  {{- if and .Values.ingress.ingressClassName (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
Shoudn't we keep validating k8s version?

Reply:
Answered in https://github.com/perses/helm-charts/pull/40#discussion_r1870667401